### PR TITLE
Blog - Building Your JavaScript Monorepo

### PR DESCRIPTION
**Pull request by msali123**
Done with all checks.

### CODE REVIEW

**Comment by msali123** [comment]
Preview URL:
https://63c97b87909dde5ef6f597fd--ephemeral-narwhal-3ac3b2.netlify.app

**Comment by msali123** [comment]
Preview URL:
https://63c987d1f5788d6ed2efc01d--ephemeral-narwhal-3ac3b2.netlify.app

File: blog/_posts/2023-02-02-building-js-monorepo.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
+---
+title: "Building Your JavaScript Monorepo"
```
Comment (jalletto) [inline blog/_posts/2023-02-02-building-js-monorepo.md:2]: @msali123 This looks great but needs one change. Somehow the title got changed on the Airtable card, so this is the wrong title. 

Title should be "Navigating the Monorepo Tools Ecosystem". Sorry for the confusion. Can you please update this and the header?

**Review by jalletto — CHANGES_REQUESTED** [review]

File: blog/_posts/2023-02-02-building-js-monorepo.md (hunk 1 above)
Comment (msali123) [reply blog/_posts/2023-02-02-building-js-monorepo.md:2]: Sure I'll change

Comment (jalletto) [reply blog/_posts/2023-02-02-building-js-monorepo.md:2]: @msali123 Actually never mind. My mistake it's fine the way it is.

**Pull request merged**